### PR TITLE
Added 50/150 word boilerplate

### PR DESCRIPTION
**Pull request by iskerrett**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/handbook/marketing/boilerplate.md [hunk 1]
```diff
@@ -15,14 +15,15 @@ FlowFuse allows organizations to reliably deliver Node-RED applications to accel
...
 
-Node-RED is very popular in the manufacturing industry for collecting factory data at the edge and transmitting it to a database or the cloud. Its visual programming environment makes it easy for non-traditional software developers to create data applications that accelerate innovation on the factory floor.
+Node-RED is very popular in the manufacturing industry for collecting factory data at the edge and transmitting it to a database or the cloud. Its visual programming environment makes it easy for non-traditional software developers to create custom applications that accelerate innovation on the factory floor.
 
-FlowFuse adds to Node-RED collaborative development, management of remote deployments, support for DevOps delivery pipelines and the ability to host Node-RED applications on FlowFuse Cloud. FlowFuse is the devops platform for Node-RED application development and delivery.
+FlowFuse and Node-RED makes it possible for manufacturing companies to build bespoke, flexible, and resilient applications that solve a business problem and drive forward transformation.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/boilerplate.md:29]: This should be the first line.

File: src/handbook/marketing/boilerplate.md [hunk 2]
```diff
@@ -15,14 +15,15 @@ FlowFuse allows organizations to reliably deliver Node-RED applications to accel
...
+
 
 ## About FlowFuse - 150 Words
 
 FlowFuse is an open-core company investing in Node-RED. Nick O’Leary, co-creator of Node-RED is FlowFuse co-founder and CTO. 
 
 Node-RED is an open-source low-code programming tool, allowing developers to quickly and easily connect IoT devices, APIs, and online services by creating flow-based applications. It uses a drag-and-drop interface to wire together nodes, which can represent devices, APIs, or functions, to process data, and create powerful automations. 
 
-Node-RED is very popular in the manufacturing industry for collecting factory data at the edge and transmitting it to a database or the cloud. Its visual programming environment makes it easy for non-traditional software developers to create data applications that accelerate innovation on the factory floor.
+Node-RED is very popular in the manufacturing industry for collecting factory data at the edge and transmitting it to a database or the cloud. Its visual programming environment makes it easy for non-traditional software developers to create custom applications that accelerate innovation on the factory floor.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/boilerplate.md:27]: I think this is too focused on the Node-RED, and not the combination of FF and NR, and blending their value. It doesn't tell the story of _why_ custom applications need to be build, and FF+NR is the _how_. The _what_ to build is for them to decide.

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/handbook/marketing/boilerplate.md (hunk 2 above)
Comment (iskerrett) [reply src/handbook/marketing/boilerplate.md]: I did an update. please take another look.

File: src/handbook/marketing/boilerplate.md (hunk 1 above)
Comment (iskerrett) [reply src/handbook/marketing/boilerplate.md]: I don't understand this comment. There is another PR that you started that change the first two sections.

File: src/handbook/marketing/boilerplate.md [hunk 3]
```diff
@@ -15,14 +15,13 @@ FlowFuse allows organizations to reliably deliver Node-RED applications to accel
 
 ## About FlowFuse - 50 Words
 
-FlowFuse helps Node-RED developers delivery applications in a more reliable, collaborative, and secure manner. Node-RED’s intuitive, low-code development environment is great for connecting together hardware devices, APIs, and online services. FlowFuse adds to Node-RED; collaborative development, management of remote deployments, support for DevOps deliver pipelines, and the ability to host Node-RED applications on FlowFuse Cloud. FlowFuse is the DevOps platform for Node-RED application development and delivery.
+FlowFuse allows manufacturers to build bespoke, flexible, and resilient applications that drive business value and transformation. FlowFuse and Node-RED provide a low-code platform that allows your engineers to quickly build applications that solve your business problems and also make it easy to adapt these applications in the future.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/boilerplate.md:18]:
```suggestion
FlowFuse empowers manufacturers to build bespoke, flexible, and resilient applications that drive business value and transformation. FlowFuse and Node-RED provide a low-code platform that allows your engineers to quickly build applications that solve your business problems and also make it easy to adapt these applications in the future.
```

Two times we use 'allow', I think this is slightly better.

File: src/handbook/marketing/boilerplate.md [hunk 4]
```diff
@@ -15,14 +15,13 @@ FlowFuse allows organizations to reliably deliver Node-RED applications to accel
...
 
-Node-RED is very popular in the manufacturing industry for collecting factory data at the edge and transmitting it to a database or the cloud. Its visual programming environment makes it easy for non-traditional software developers to create data applications that accelerate innovation on the factory floor.
-
-FlowFuse adds to Node-RED collaborative development, management of remote deployments, support for DevOps delivery pipelines and the ability to host Node-RED applications on FlowFuse Cloud. FlowFuse is the devops platform for Node-RED application development and delivery.
+FlowFuse is the platform that allows manufacturing companies to build custom applications that fit their unique requirements. Off the shelf software doesn't work in the manufacturing industry. Each manufacturing operation is unique so you need a tool platform that allows your engineers to easily build custom applications that fit your unique requirements. FlowFuse and Node-RED combined together is the platform that allows manufacters to build bespoke, flexible and resilient applications that solve their business requirements.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/boilerplate.md:27]:
```suggestion
FlowFuse is the platform that empowers manufacturing companies to build custom applications that fit their unique requirements. Off the shelf software doesn't work in the manufacturing industry. Each manufacturing operation is unique so you need a tool platform that allows your engineers to easily build custom applications that fit your unique requirements. FlowFuse and Node-RED combined together is the platform that allows manufactures to build bespoke, flexible and resilient applications that solve their business requirements.
```

Comment (ZJvandeWeg) [reply src/handbook/marketing/boilerplate.md:27]: I cannot leave notes on lines that didn't change with a suggestion, so I think this should be ordered differently:

```
FlowFuse is the platform that allows manufacturing companies to build custom applications that fit their unique requirements. Off the shelf software doesn't work in the manufacturing industry. Each manufacturing operation is unique so you need a tool platform that allows your engineers to easily build custom applications that fit your unique requirements. FlowFuse and Node-RED combined together is the platform that allows manufacters to build bespoke, flexible and resilient applications that solve their business requirements.

FlowFuse is an open-core company investing in Node-RED. Nick O’Leary, co-creator of Node-RED is FlowFuse co-founder and CTO. 

Node-RED is an open-source low-code programming tool, allowing developers to quickly and easily connect IoT devices, APIs, and online services by creating flow-based applications. It uses a drag-and-drop interface to wire together nodes, which can represent devices, APIs, or functions, to process data, and create powerful applications.
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Pull request merged**